extensions/mssql/src/reactviews/common/locConstants.ts

```diff
@@ -1312,6 +1312,60 @@ export class LocConstants {
             noDataAvailable: l10n.t("No data available"),
             noPendingChanges: l10n.t("No pending changes. Make edits to generate a script."),
             closeScriptPane: l10n.t("Close Script Pane"),
+            filterContains: l10n.t("Contains"),
+            filterNotContains: l10n.t("Not contains"),
+            filterEquals: l10n.t("Equals"),
+            filterNotEqualTo: l10n.t("Not equal to"),
+            filterStartsWith: l10n.t("Starts With"),
```

Comment:
I think all of these strings should be label-cased:

* "Starts With" -> "Starts with"
* "Clear all Filters" -> "Clear all filters"

etc.